Exercise_1/index.html

```diff
@@ -0,0 +1,128 @@
+                popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and
+                more recently with desktop publishing software like Aldus PageMaker including versions of Lorem Ipsum.
+
+                Lorem Ipsum is simply dummy text of the printing and typesetting industry.
+                Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer
+                took a galley of type and scrambled it to make a type specimen book. It has survived not only five
+                centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was
+                popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and
+                more recently with desktop publishing software like Aldus PageMaker including versions of Lorem Ipsum.
+            </article>
+            <img src="img/images.png"
+                 alt="Cracow" width="100" height="100" align="right"/>
+            <article>
+
+                Lorem Ipsum is simply dummy text of the printing and typesetting industry.
+                Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown
+                printer
+                took a galley of type and scrambled it to make a type specimen book. It has survived not only
+                five
+                centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It
+                was
+                popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages,
+                and
+                more recently with desktop publishing software like Aldus PageMaker including versions of Lorem
+                Ipsum.
+
+                Lorem Ipsum is simply dummy text of the printing and typesetting industry.
+                Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown
+                printer
+                took a galley of type and scrambled it to make a type specimen book. It has survived not only
+                five
+                centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It
+                was
+                popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages,
+                and
+                more recently with desktop publishing software like Aldus PageMaker including versions of Lorem
+                Ipsum.
+                <sup>2</sup>
+
+                <div>
+                    <br>
+                    1.Polish prononuciation ......................................................
+                    <br>
+                    2. Information source:Wikipedia
+                </div>
+
+                <h3>Tonight in Kraków:</h3>
+
+                <table width="100%" align="center">
```

Comment:
Remove `align="center"`, add `<thead>`, add `<tbody align="center">` and remove `align="center"` from everywhere.